android_do

```diff
@@ -95,35 +98,38 @@ fi
 
 rm cjdroute
 
-#x86 build
-rm -rf build_linux
-
-export CROSS_COMPILE=$BUILD_PATH/i686-${TYPE}-androideabi/bin/i686-linux-android-
-export TARGET_ARCH=x64
-export CROSS=${CROSS_COMPILE}
-export CC=${CROSS}gcc
-export AR=${CROSS}ar
-export RANLIB=${CROSS}ranlib
-export CFLAGS=${CROSS_CFLAGS}
-export LDFLAGS=${CROSS_LDFLAGS}
-gcc_version=$(${CC} --version)
-echo Using $gcc_version
-echo Compiler CC: $CC
-echo Compiler CFLAGS: $CFLAGS
-echo Compiler LDFLAGS: $LDFLAGS
-time ./do
-
-cp cjdroute $(pwd)/android_out/x86/ || ret=$?
-
-if [ "$ret" != "" ] && [ "$ret" != "0" ]; then
-  echo -e "\e[1;31mCopying x86 binary failed, non zero status returned - $ret\e[0m"
-else
-  echo -e "\e[1;32mCopied x86 successfully\e[0m"
-fi
-
-rm cjdroute
-
-echo -e "\n\e[1;34mOutput: $(pwd)/android_out/armeabi-v7a/cjdroute\e[0m"
-echo -e "\e[1;34m        $(pwd)/android_out/x86/cjdroute\e[0m"
+# NOTE: Does not currently build on x86. It seems that the build scripts only build libuv for arm
```

Comment:
Currently the libuv gyp scripts only compile arm so as far as I can tell the x86 build is already broken. @benhylau can you verify that the arm build works now? Thanks!